pmm-app/src/shared/components/Form/NumericInput/NumericInput.style.ts

```diff
@@ -0,0 +1,101 @@
+import { css } from 'emotion';
+// import { GrafanaTheme } from '@grafana/data'
+import { stylesFactory } from '@grafana/ui';
+
+// export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+export const getStyles = stylesFactory((theme) => {
```

Comment:
I think you can keep the GrafanaTheme type here and remove the comments. If it gives you an error because it doesn't find the colors just do `const { colors }: any = theme;` and use the theme type for other variables (e.g. spacing).

Reply:
Good catch! Actually it's not giving any errors, it was just a bad copy-paste from the JS sandbox to TS :)